node-commons/src/main/java/org/apache/iotdb/commons/consensus/ConsensusGroupId.java

```diff
@@ -80,5 +80,9 @@ public static ConsensusGroupId createFromTConsensusGroupId(
         TConsensusGroupId tConsensusGroupId) {
       return create(tConsensusGroupId.getType().getValue(), tConsensusGroupId.getId());
     }
+
+    public static TConsensusGroupId convertToTConsensusGroupId(ConsensusGroupId consensusGroupId) {
```

Comment:
we can directly add this function to `ConsensusGroupId` default class function just like `HashCode()`?

Reply:
Fixed.